test/EzMap.IntegrationTest/PoiControllerTest.cs

```diff
@@ -99,4 +99,67 @@ public async Task DeletePoi_CorrectDataProvided_PoiShouldBeDeleted()
         Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         // assert
     }
+
+    [Fact]
+    public async Task GetPoi_GuidProvided_DetailOfPoiShouldBeFound()
+    {
+        var app = new TestWebAppFactory<Program>();
+        var client = app.CreateClient();
+        using var scope = app.Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<EzMapContext>();
+        var token = await TestHelper.GetDefaultUserToken(client);
+
+
+        var poi = new Poi("home", "59 ntt", TestUser.DefaultUser.Id);
+        dbContext.Pois.Add(poi);
+
+        await dbContext.SaveChangesAsync();
+        
+        // act
+        using var response = await client.RequestAsJsonAsyncWithToken<object>(HttpMethod.Get, $"api/poi/{poi.Id}", token);
+        response.EnsureSuccessStatusCode();
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        
+        var responsePoi = JsonSerializer.Deserialize<Poi>( await response.Content.ReadAsStringAsync() , new JsonSerializerOptions()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        });
+        
+        Assert.Equal("home", responsePoi.Name);
```

Comment:
should not use hardcoded string here, consider `Assert.Equal(poi.Name, responsePoi.Name);`
Same for every assert